### DISCUSSION TITLE
Welcome to LegacyGwent Discussions!

### DISCUSSION THREAD

**Discussion by DeusSeuca** [LegacyGwent/LegacyGwent#331] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NaZ0707** [comment]
how do i download it on iOS using Xcode since i can't find an ipa file ... i am new to this so any help would be appreciated

**Comment by Pippow** [comment]
@DeusSeuca Hello, can you please restart the server?
